main/ertp/api/purse.md

```diff
@@ -1,78 +1,126 @@
+with the `brand` of assets you want the new `purse` to hold. It returns the
+new `purse` with an empty balance. 
+```js
+const {issuer: quatloosIssuer} = makeIssuerKit('quatloos');
+quatloosPurse1 = quatloosIssuer.makeEmptyPurse();
+quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
+```
 
 ## purse.getCurrentAmount()
 - Returns: `{Amount}`
 
-Get the current `amount` contained in this purse, confirmed by the
-issuer. Note that this amount will change from call to call if assets
-have been deposited or withdrawn between calls. 
+Get an `amount` describing the current digital assets balance in the `purse`.
+This `amount` is confirmed by the `issuer` for the `brand` the `purse` can hold.
+Of course, the returned `amount` `value` might be different the next time you
+call `getCurrentAmount()` on the same `purse` if assets have been deposited or
+withdrawn from it in-between calls. . 
 
 ```js
-const { issuer } = makeIssuerKit('bucks');
-const purse = issuer.makeEmptyPurse();
-
-const currentBalance = purse.getCurrentAmount();
+const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+// quatloos5 is a payment with balance of 5 quatloos
+const quatloosPurse.deposit(quatloos5);
+// Returns an amount with value = 5 and brand = quatloos
+const currentBalance = quatloosPurse.getCurrentAmount();
 ```
 
 ## purse.deposit(payment, optAmount)
 - `payment` `{Payment}`
-- `optAmount` `{Amount}` - Optional. This parameter ensures you are depositing the amount you expect.
+- `optAmount` `{Amount}` - Optional. 
 - Returns: `{Amount}`
 
-Deposit all the contents of `payment` into this purse, returning the payment's amount
-of digital assets (i.e. the deposit amount). If the optional argument `optAmount` does not equal the balance of
-`payment`, or if `payment` is an unresolved promise, throw an error.
+Deposit all the contents of `payment` into this `purse`, returning an `amount` describing the
+`payment`'s digital assets (i.e. the deposit amount). If the optional argument `optAmount` does not equal the balance of
+`payment`, or if `payment` is an unresolved promise, it throws an error.
 
 ```js
-const { issuer, mint, amountMath } = makeIssuerKit('bucks');
-const purse = issuer.makeEmptyPurse();
-const payment = mint.mintPayment(amountMath.make(123));
-const bucks123 = amountMath.make(123);
+const { issuer: quatloosIssuer, mint: quatloosMint, amountMath: quatloosAmountMath } = 
+      makeIssuerKit('quatloos');
+const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+const payment = quatloosMint.mintPayment(quatloosAmountMath.make(123));
+const quatloos123 = quatloosAmountMath.make(123);
 
-// Deposit a payment for 123 bucks into the purse. Ensure that this is the amount you expect.
-purse.deposit(payment, bucks123);
+// Deposit a payment for 123 Quatloos into the purse. Ensure that this is the amount you expect.
+quatloosPurse.deposit(payment, quatloos123);
```

Comment:
```suggestion
const depositAmount = quatloosPurse.deposit(payment, quatloos123);
```

One has to look at the return value to find out if the deposit succeeded. I dunno if you want to add a line of code checking that the returned value was 123.